src/dewarping/DetectVertContentBounds.cpp

```diff
@@ -247,7 +247,7 @@ QLineF SequentialColumnProcessor::approximateWithLine(std::vector<Segment>* dbgS
   // Run RANSAC on the segments.
 
   RansacAlgo ransac(segments);
-#if QT_VERSION_MAJOR == 5 and QT_VERSION_MINOR < 14
+#if QT_VERSION_MAJOR == 5 && QT_VERSION_MINOR <= 9
```

Comment:
Is `<=9` instrad of `<14` really intended here?

Reply:
Hi @vigri .

@plzombie specifically pointed out just such a version in the given condition.